### ISSUE TITLE
adversarial finetuning recipe for downstream datasets?

### ISSUE THREAD

**Issue by HashmatShadab** [nmndeep/revisiting-at#4]
Can you please provide in more detail the training recipe used for adversarial finetuning on downstream datasets(cifar10, cifar1000, flowers)? What optimizer, augmentations are used? Is the adversarial training done similar to how it is done on ImageNet or TRADES framework is used?

**Comment by nmndeep** [comment]
Hi, The main part of training recipe for downstream tasks can already be found in App. C.4.
We use AdamW as optimizer and train with regular CE loss (similar to ImageNet training). 
We have never used TRADES in our work.
Hope this helps.

**Comment by HashmatShadab** [comment]
Thank you!
